app/sidero-controller-manager/internal/ipxe/ipxe_server.go

```diff
@@ -78,6 +78,7 @@ var (
 	apiPort                   int
 	extraAgentKernelArgs      string
 	defaultBootFromDiskMethod BootFromDisk
+	sideroLinkConfig          siderolink.Config
```

Comment:
I removed that now in favor of `siderolink.Cfg` which gets initialized as part of the controller bootstrap

Reply:
yeah, that's how I was setting up in my PR and I forgot to remove it.